nimpretty/tests/expected/exhaustive.nim

```diff
@@ -486,132 +486,132 @@ type
     HttpPatch        ## Applies partial modifications to a resource.
 
 type
-  HtmlTag* = enum   ## list of all supported HTML tags; order will always be
-                    ## alphabetically
-    tagUnknown,     ## unknown HTML element
-    tagA,           ## the HTML ``a`` element
-    tagAbbr,        ## the deprecated HTML ``abbr`` element
-    tagAcronym,     ## the HTML ``acronym`` element
-    tagAddress,     ## the HTML ``address`` element
-    tagApplet,      ## the deprecated HTML ``applet`` element
-    tagArea,        ## the HTML ``area`` element
-    tagArticle,     ## the HTML ``article`` element
-    tagAside,       ## the HTML ``aside`` element
-    tagAudio,       ## the HTML ``audio`` element
-    tagB,           ## the HTML ``b`` element
-    tagBase,        ## the HTML ``base`` element
-    tagBdi,         ## the HTML ``bdi`` element
-    tagBdo,         ## the deprecated HTML ``dbo`` element
-    tagBasefont,    ## the deprecated HTML ``basefont`` element
-    tagBig,         ## the HTML ``big`` element
-    tagBlockquote,  ## the HTML ``blockquote`` element
-    tagBody,        ## the HTML ``body`` element
-    tagBr,          ## the HTML ``br`` element
-    tagButton,      ## the HTML ``button`` element
-    tagCanvas,      ## the HTML ``canvas`` element
-    tagCaption,     ## the HTML ``caption`` element
-    tagCenter,      ## the deprecated HTML ``center`` element
-    tagCite,        ## the HTML ``cite`` element
-    tagCode,        ## the HTML ``code`` element
-    tagCol,         ## the HTML ``col`` element
-    tagColgroup,    ## the HTML ``colgroup`` element
-    tagCommand,     ## the HTML ``command`` element
-    tagDatalist,    ## the HTML ``datalist`` element
-    tagDd,          ## the HTML ``dd`` element
-    tagDel,         ## the HTML ``del`` element
-    tagDetails,     ## the HTML ``details`` element
-    tagDfn,         ## the HTML ``dfn`` element
-    tagDialog,      ## the HTML ``dialog`` element
-    tagDiv,         ## the HTML ``div`` element
-    tagDir,         ## the deprecated HTLM ``dir`` element
-    tagDl,          ## the HTML ``dl`` element
-    tagDt,          ## the HTML ``dt`` element
-    tagEm,          ## the HTML ``em`` element
-    tagEmbed,       ## the HTML ``embed`` element
-    tagFieldset,    ## the HTML ``fieldset`` element
-    tagFigcaption,  ## the HTML ``figcaption`` element
```

Comment:
Bah, I prefer the old way here. :-(

Reply:
There are two problems with the old way:

1. When there is no comment alignment, it produces an extra space which humans wouldn't do:
```nim
if true:  # extra space
  echo 7
```

2. When there is an alignment, such as an example you picked above, it is not consistent — the number of spaces depends on whether you use commas for enums or not:

```nim
type
  HtmlTag* = enum       ## list of all supported HTML tags; order will always be
                        ## alphabetically
    tagBasefont,        ## the deprecated HTML ``basefont`` element
    tagBig,             ## the HTML ``big`` element
    tagBlockquoteLong,  ## the HTML ``blockquote`` element
    tagBody,            ## the HTML ``body`` element
    tagBr,              ## the HTML ``br`` element
    tagButton           ## the HTML ``button`` element


type
  HtmlTag* = enum     ## list of all supported HTML tags; order will always be
                      ## alphabetically
    tagBasefont       ## the deprecated HTML ``basefont`` element
    tagBig            ## the HTML ``big`` element
    tagBlockquoteLong ## the HTML ``blockquote`` element
    tagBody           ## the HTML ``body`` element
    tagBr             ## the HTML ``br`` element
    tagButton         ## the HTML ``button`` element
```

-----

This PR solves both of those cases.

Reply:
Alright.